packages/schematics/src/collection/ngrx/files/__directory__/__fileName__.selectors.ts__tmpl__

```diff
@@ -0,0 +1,24 @@
+import { createFeatureSelector, createSelector } from '@ngrx/store';
```

Comment:
What does the ngrx/schematics package generates for selectors? Something similar?

Reply:
No selectors are generated in the current version. I believe Brandon has been playing with this idea.